api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -28,6 +28,14 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Role Based Access Control (RBAC)]
 
+enum MetadataSource {
+  // Query :ref:`dynamic metadata <well_known_dynamic_metadata>`
+  DYNAMIC = 0;
+
+  // Query :ref:`route entry metadata <envoy_v3_api_field_config.route.v3.Route.metadata>`
+  ROUTE_ENTRY = 1;
```

Comment:
`ROUTE`, to keep consistance with our %METADATA%

Reply:
I was following the RateLimit use-case which is calling it [ROUTE_ENTRY](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#enum-config-route-v3-ratelimit-action-metadata-source). Should we keep this name or change it to `ROUTE`?